index.html

```diff
@@ -8,9 +8,9 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container" style="background-image:url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');">
+    <div class="container">
         <div class="toolbar">
-            <p class="heading-1" style="color:blue ;">My logo</p>
+            <p class="heading">My logo</p>
         </div>
         <div>
```

Comment:
- [ ] kindly consider using a `section` tag here for goof semantics as suggested by the previous reviewers.